src/main/java/ch/njol/skript/expressions/ExprTimePlayed.java

```diff
@@ -82,12 +83,9 @@ public Timespan convert(Player player) {
 	
 	@Override
 	public void change(Event e, @Nullable Object[] delta, Changer.ChangeMode mode) {
```

Comment:
I know it's kind of unrelated but while we're here, can you directly import ChangeMode